source/howtos/run-sync-1.5.rst

```diff
@@ -140,7 +140,7 @@ non-Mozilla-hosted Sync server. **Before logging in**, go to App Menu > Settings
 > About Firefox and click the logo 5 times. You should see a "debug menu enabled" 
 notification. Go back to the main menu and you will see two options for a custom 
 account server and a custom Sync server. Set the Sync server to the URL given 
-above and then log in.
+above (in the format of "http://localhost:5000/token/1.0") and then log in.
```

Comment:
That surprises me - the code that fixes up the URL is [here](https://github.com/mozilla/application-services/blob/3567a6d920165501897ba31a6346d1177fcdd3d3/components/sync15/src/token.rs#L66) - so it looks like the change you made would explicitly not work. What's the final URL to your token server?